requirements.txt

```diff
@@ -9,3 +9,4 @@ requests
 flask
 pygments
 litecli>=1.3.0
+lief
```

Comment:
Ideally we should pin this to a major version (I'm going to do that with the rest of the dependencies soon, fwiw).